extensions/resource-deployment/src/services/notebookService.ts

```diff
@@ -34,6 +34,7 @@ export interface NotebookExecutionResult {
 
 export interface INotebookService {
 	launchNotebook(notebook: string | NotebookInfo): Thenable<azdata.nb.NotebookEditor>;
+	insertCellAndLaunchNotebook(notebook: string | NotebookInfo, cellStatements: string[], insertionPosition?: number): Promise<void>;
```

Comment:
you can still use launchNotebook as the method name, just add an override to it.